### PR TITLE
Beta1.12.1rc0  updates since waternet fixes 

### PR DESCRIPTION
**Pull request by afostr**
created for review purposes

### CODE REVIEW

File: src/p2p/Active.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ let p2pLogger: Logger
 
 let activeRequests: Map<P2P.NodeListTypes.Node['publicKey'], P2P.ActiveTypes.SignedActiveRequest>
 let queuedRequest: P2P.ActiveTypes.ActiveRequest
-let neverGoActive = false
+export let neverGoActive = false
```
Comment (mssabr01) [inline src/p2p/Active.ts:100]: Should go in the debug config in server.ts

File: src/p2p/Lost.ts [hunk 2]
```diff
@@ -551,6 +549,17 @@ export function parseRecord(record: P2P.CycleCreatorTypes.CycleRecord): P2P.Cycl
     }
   }
 
+  // If we see our node in the lostSyncing field of the cycle record, emit a sync timeout event
+  for (const id of record.lostSyncing) {
+    const node = NodeList.nodes.get(id)
+    if (node == null) {
+      error(`Lost syncing node ${id} is not in the network`)
```
Comment (mssabr01) [inline src/p2p/Lost.ts:556]: This comment says emit an event if our node is found in the lostSyncing record, but this is emitting an event for every node found

Comment (mssabr01) [reply src/p2p/Lost.ts:556]: Is the comment inaccurate? This is part of the slashing logic and every node that is part of an emitted event here gets a slashing tx created for it. Perhaps the comment is backwards.

**Review by mssabr01 — APPROVED** [review]

File: src/p2p/Active.ts (hunk 1 above)
Comment (afostr) [reply src/p2p/Active.ts:100]: i agree .  created blue 204 to fix in a following PR

File: src/p2p/Lost.ts (hunk 2 above)
Comment (afostr) [reply src/p2p/Lost.ts:556]: comments were wrong . I have confirmed and clarified in my latest branch

**Comment by afostr** [comment]
this PR was used to help us review partial work.  it is now merged in so closing

**Pull request closed without merging**